views/verifyEmailSuccess.pug

```diff
@@ -0,0 +1,9 @@
+include include/sauce-common
+
++sauce-layout('Your request has been submitted!')(hideNavbar, includeBootstrapScripts)
+  h1.mb-4 We've sent you an email, follow the instructions to complete setting up your SCCS account
+  p.
+    To prevent abuse, SCCS staff manually approve all account creation 
```

Comment:
This isn't true anymore with these new changes!

(But also, don't worry about this, I'll write some copy here. This is more for my own note)